pkg/utils/comparators/daemonsets/comparator.go

```diff
@@ -1,66 +1,64 @@
 package daemonsets
 
 import (
+	"github.com/openshift/cluster-logging-operator/pkg/log"
 	"reflect"
 
-	"github.com/openshift/cluster-logging-operator/pkg/logger"
 	"github.com/openshift/cluster-logging-operator/pkg/utils"
 	apps "k8s.io/api/apps/v1"
 )
 
 //AreSame compares daemonset for equality and return true equal otherwise false
 func AreSame(current *apps.DaemonSet, desired *apps.DaemonSet) bool {
-	logger.Tracef("Comparing DaemonSets current %v to desired %v", current, desired)
-
 	if !utils.AreMapsSame(current.Spec.Template.Spec.NodeSelector, desired.Spec.Template.Spec.NodeSelector) {
-		logger.Debugf("DaemonSet %q nodeSelector change", current.Name)
+		log.Info("DaemonSet nodeSelector change", "DaemonSetName", current.Name)
```

Comment:
this should really be debug

Reply:
there won't be debug, if it doesn't make sense to have it at Info or Error, consider what sort of use it serves if any

Reply:
It really looks like debug info, but if it is not supported, can I remove this lines at all?

Reply:
We have verbosity levels: the default level should be logs intended for an admin running in a production environment. I suggest for now we make everything else log.V(9).Info(). We can refine our use of V levels later if we need to.

Reply:
I think we should probably establish a convention and I'll document that convention in github.com/ViaQ/logerr. My suggestion is to ask yourself "On a scale from 1-5 (5 being noisiest), how noisy is this message?" You don't really have to think too hard about it. It's similar to picking scrum points. Compare it to another logline and decide if it's more than, less than, or equal to that line. I would hope that in future PRs It shouldn't be much of a conversation piece unless the log rate is fundamentally misunderstood. I suggest we keep it very simple and avoid any kind of inertia caused by uncertainty. By keeping the range small (1-5) we can easily pick a number without vacillation. 

@alanconway what do you think?

Reply:
Sounds good to me. Since these messages are for developers (us) we have freedom to adjust levels based on debugging experience when something is too noisy/not noisy enough.